dist/index.html

```diff
@@ -0,0 +1,146 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>HTML Form</title>
+        <link rel="stylesheet" href="/style.e308ff8e.css">
+    <script src="/style.e308ff8e.js"></script></head>
+    <body>
+        <script type="text/javascript" src="/main.1f19ae8e.js"></script>
+        <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" class="form">
+
+            <fieldset class="group">
+                <legend>
+                  Personal information
+                </legend>
+                <div class="simple">
+                    <label for="sname">
+                      Surname:
+                        <input type="text" name="sname" id="sname" required="" autocomplete="off">
+                    </label>
```

Comment:
Fixes due to previous mentor's recommendation
1) children should be at one level according to code styling recommendations [read point 2](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
2) attributes [read point 3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
                    <label for="sname">
                      Surname:
                      <input 
                        type="text" 
                        name="sname" 
                        id="sname" 
                        required 
                        autocomplete="off"
                      >
                    </label>
```